server/sonar-server/src/test/java/org/sonar/server/plugins/UpdateCenterMatrixFactoryTest.java

```diff
@@ -31,14 +31,14 @@
 
 public class UpdateCenterMatrixFactoryTest {
```

Comment:
It would have been interesting to improve these UTs, for instance to reflect what has been done in this PR